NEWS

```diff
@@ -1,3 +1,40 @@
+26-Jan-2017 IGNITION v0.21.0
+
+  Features
+
+    - Add support for networkd drop-ins
+    - Add new binary, ignition-validate, for validating Ignition configs
+    - Add overwrite field to files, directories, and links for deleting
+      preexisting items at the node's path
+    - Add options field to raid for specifying arbitrary mdadm options
+    - Add append field to files for appending to preexisting files
```

Comment:
The first `files` refers to the `files` section, and the second refers to actual files.  Without that context, this could be confusing.